python/ray/util/client/server/dataservicer.py

```diff
@@ -48,8 +48,17 @@ def Datapath(self, request_iterator, context):
                 else:
                     assert accepted_connection
```

Comment:
what do you expect this assert to do?

Reply:
@AmeerHajAli I did not add this. But it just ensures `init` has been called before this Datapath is useed.